utils/githooks/pre-commit.d/30-Jenkinsfile.sh

```diff
@@ -1,6 +1,7 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #
 #  Copyright 2023-2024 Intel Corporation.
```

Comment:
Is there any specific reason why Intel copyright years differ on each file? It maybe related to the file creation date... Still seems little confusing... some files have 2024 alone, 2022-2024, 2023-2024, etc...

Reply:
That is intended. Copyright on a file begins when it is created